build.override.targets

```diff
@@ -3,37 +3,39 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
+    <ScenarioLatestDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\stable.dependencies.json</ScenarioLatestDependencyJson>
+    <UseStableDependencies Condition="'$(UseStableDependencies)'==''">true</UseStableDependencies>
```

Comment:
I think the way I would prefer to see this done would be have a property for the location of the preferred json file based on some condition.  
And the final pick of stable would occur if nothing else had set it.
This allows us future ways to add other kinds of dependencies (like $(UseLKGDependencies)) and also allows us to name a .json file to use from the command line.

```
    <PropertyGroup>
      <UseLatestDependencies Condition="'$(UseLatestDependencies)'==''">false</UseLatestDependencies>
    </PropertyGroup>
    
    <PropertyGroup Condition = "'$(UseLatestDependencies)' == 'true' >
        <ScenarioDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\latest.dependencies.json</ScenarioDependencyJson> 
    </PropertyGroup>
        
    <PropertyGroup Condition = "'$(ScenarioDependencyJson)' == '' >
        <ScenarioDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\stable.dependencies.json</ScenarioDependencyJson> 
    </PropertyGroup>

```